src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -331,8 +346,73 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
     response.toString.parseJson.asJsObject
   }
 
+  override def trialBillingManagerRemoveBillingAccount(project: String, targetUserEmail: String): Boolean = {
+    val projectName = s"projects/$project" // format needed by Google
+
+    // as the user, get the current billing info to make sure we are removing the right thing.
+    // this call will fail if the user doesn't have permissions on the project (this is a doublecheck
+    // that the user is a member of the project)
+    // val billingService = getCloudBillingManager(getTrialBillingManagerCredential(Some(targetUserEmail)))
```

Comment:
this isn't succeeding, so I have it commented out. I also tried using the cloud resource manager api to query the project directly, but that's also failing.